nxviz/utils.py

```diff
@@ -121,6 +123,25 @@ def num_discrete_groups(data_container):
     return len(set(data_container))
 
 
+def itemes_in_groups(data_container):
```

Comment:
I think the function name might be misspelled here - should probably be `items_in_groups`?